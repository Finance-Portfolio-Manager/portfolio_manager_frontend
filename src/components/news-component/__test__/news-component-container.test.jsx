import React, { Component } from 'react';
import ReactDOM from 'react-dom';
import NewsContainer from './../news-component-container';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

import { shallow, ShallowWrapper, mount } from "enzyme";

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
const wrapper = shallow(<NewsContainer></NewsContainer>);

test("matches snapshot", ()=>{
    const tree = renderer.create(<NewsContainer></NewsContainer>).toJSON();
    expect(tree).toMatchSnapshot();
})