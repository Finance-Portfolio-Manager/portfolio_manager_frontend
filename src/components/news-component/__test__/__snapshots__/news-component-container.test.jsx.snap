// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="w-25 card"
>
  <div
    className="w-100 card-body"
  >
    <div
      className="carousel slide"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
    >
      <ol
        className="carousel-indicators"
      >
        <li
          className="active"
          onClick={[Function]}
        />
        <li
          className=""
          onClick={[Function]}
        />
        <li
          className=""
          onClick={[Function]}
        />
        <li
          className=""
          onClick={[Function]}
        />
        <li
          className=""
          onClick={[Function]}
        />
      </ol>
      <div
        className="carousel-inner"
      >
        <div
          className="carousel-item active"
        >
          <a
            href="https://google.com"
          >
            <img
              alt="Slide 1"
              className="w-100"
              src="https://images.unsplash.com/photo-1590283603385-17ffb3a7f29f?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1800&q=80"
              style={
                Object {
                  "height": 360,
                }
              }
            />
          </a>
          <div
            className="carousel-caption d-none d-md-block"
          >
            <h3>
              Slide 1
            </h3>
            <p>
              Slide 1
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <a
            href="https://google.com"
          >
            <img
              alt="Slide 2"
              className="w-100"
              src="https://images.unsplash.com/photo-1611974789855-9c2a0a7236a3?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80"
              style={
                Object {
                  "height": 360,
                }
              }
            />
          </a>
          <div
            className="carousel-caption d-none d-md-block"
          >
            <h3>
              Slide 2
            </h3>
            <p>
              Slide 2
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <a
            href="https://google.com"
          >
            <img
              alt="Slide 3"
              className="w-100"
              src="https://images.unsplash.com/photo-1583752028088-91e3e9880b46?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80"
              style={
                Object {
                  "height": 360,
                }
              }
            />
          </a>
          <div
            className="carousel-caption d-none d-md-block"
          >
            <h3>
              Slide 3
            </h3>
            <p>
              Slide 3
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <a
            href="https://google.com"
          >
            <img
              alt="Slide 4"
              className="w-100"
              src="https://images.unsplash.com/photo-1583752028088-91e3e9880b46?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80"
              style={
                Object {
                  "height": 360,
                }
              }
            />
          </a>
          <div
            className="carousel-caption d-none d-md-block"
          >
            <h3>
              Slide 4
            </h3>
            <p>
              Slide 4
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <a
            href="https://google.com"
          >
            <img
              alt="Slide 5"
              className="w-100"
              src="https://images.unsplash.com/photo-1583752028088-91e3e9880b46?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80"
              style={
                Object {
                  "height": 360,
                }
              }
            />
          </a>
          <div
            className="carousel-caption d-none d-md-block"
          >
            <h3>
              Slide 5
            </h3>
            <p>
              Slide 5
            </p>
          </div>
        </div>
      </div>
      <a
        className="carousel-control-prev"
        onClick={[Function]}
        role="button"
        style={
          Object {
            "cursor": "pointer",
          }
        }
        tabIndex="0"
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="sr-only"
        >
          prev
        </span>
      </a>
      <a
        className="carousel-control-next"
        onClick={[Function]}
        role="button"
        style={
          Object {
            "cursor": "pointer",
          }
        }
        tabIndex="0"
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="sr-only"
        >
          next
        </span>
      </a>
    </div>
  </div>
</div>
`;
